tests/e2e-vcluster/vcluster-ops/setup-vdb/base/setup-vdb.yaml

```
@@ -0,0 +1,41 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertica.com/v1beta1
+kind: VerticaDB
+metadata:
+  name: verticadb-sample
```

This test needs to set the feature flag that enables vcluster ops. Lets add the following annotation.
```suggestion
  name: verticadb-sample
  annotations:
    vertica.com/vcluster-ops: "true"
```